client/templates/initiative/card/initiativeCard.js

```diff
@@ -12,5 +22,13 @@ Template.initiativeCard.events({
     e.preventDefault();
     var initiative = this;
     addOrRemoveVote(initiative); 
+  },
+  'click .status': function(e, tpl){
+    Meteor.call('setInactiveActive', this, function(err, res){
+      if(err)
+        sAlert.error(err.message);
+
```

Comment:
probably want an else here, so we don't show an error and say it was updated at the same time.